File: python/v1/demo_get_pid.py
import dh
from dh_logger import logger

def main():
    pid_p = dh.get_p()
    pid_i = dh.get_i()
    pid_d = dh.get_d()

    for i in range(0, 6):
        logger.print_trace("id: {}, p: {}, i: {}, d: {}".format(i, pid_p[i], pid_i[i], pid_d[i]))



if __name__ == '__main__':
    main()

File: fdh/c++/v1/source/demo_get_pid.cpp
#include <iostream>
#include "dh.h"

using namespace DH;
using namespace Predefine;
using namespace Utils;

int main()
{
    Logger::get_instance()->print_trace("p: %f\n", get_p(RIGHT_HAND, FORE_F));
    Logger::get_instance()->print_trace("i: %f\n", get_i(RIGHT_HAND, FORE_F));
    Logger::get_instance()->print_trace("d: %f\n", get_d(RIGHT_HAND, FORE_F));
    return 0;
}